docs/self-managed/zeebe-deployment/configuration/gateway.md

```diff
@@ -208,6 +208,26 @@ membership:
   syncInterval: 10s
 ```
 
+### zeebe.gateway.cluster.configManager.gossip
+
+Configure the parameters used to propagate the dynamic cluster configuration across brokers and gateways.
+
+| Field              | Description                                                                                                                                                                                                    | ExampleValue |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| syncDelay          | Sets the interval between two synchronization requests to other members of the cluster. This setting can also be overridden using the environment variable ZEEBE_BROKER_CLUSTER_CONFIGMANAGER_GOSSIP_SYNCDELAY | 10s          |
```

Comment:
This should be `ZEEBE_GATEWAY_CLUSTER..`, right?

Reply:
Yes, forgot to change after pasting :+1: